### PR TITLE
Varvara/feature/signin signup

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: src/components/layouts/sign_up_sp_layout/SignUpSPLayout.stories.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { ComponentMeta, ComponentStory } from '@storybook/react';
+import SignUpStartPage from '../../../components/forms/sign_up_start_page/SignUpStartPage';
```
Comment (kipod) [inline src/components/layouts/sign_up_sp_layout/SignUpSPLayout.stories.tsx]: use absolute path from project root

File: src/pages/index.tsx [hunk 2]
```diff
@@ -5,12 +5,12 @@ export default function Home() {
...
       <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
+        <title>Find My Coach</title>
+        <meta name="description" content="Find My Coach" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <main className={styles.main}>Hello</main>
+      <main className={styles.main}>Hello !!!!</main>
```
Comment (kipod) [inline src/pages/index.tsx]: 4 (!) ?

File: src/pages/sign_up/coach_student.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import Head from 'next/head';
+import linkLogo from '../../../public/fmc_logo.png';
+import SignUpStartPage from '../../components/forms/sign_up_start_page/SignUpStartPage';
+import SignUpSPLayout from '../../components/layouts/sign_up_sp_layout/SignUpSPLayout';
```
Comment (kipod) [inline src/pages/sign_up/coach_student.tsx]: prefer absolute paths....

File: src/pages/sign_up/coach_student.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <>
+      <Head>
+        <title>Sign Up Coach Student</title>
+        <meta name="description" content="Sign Up Coach Student" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <SignUpSPLayout
+        linkBackgroundImg={'../../../img/young-basketball-bg.png'}
```
Comment (kipod) [inline src/pages/sign_up/coach_student.tsx:17]: use an absolute path, but not relative

File: src/styles/Home.module.sass [hunk 5]
```diff
@@ -1,4 +1,4 @@
 @import "variables"
 
 .main
-  background-color: $primary-color
\ No newline at end of file
+  background-color: $color-sign-sp-layout
```
Comment (kipod) [inline src/styles/Home.module.sass]: add EOL in EOF

File: src/styles/_mixins.sass [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    width: 100vw
+    height: 100vh
+    z-index: 1
+    opacity: 0.75
+    background-color: $color
+
+@mixin box-center($justifyContent)
+    display: flex
+    justify-content: $justifyContent
+    align-items: center
```
Comment (kipod) [inline src/styles/_mixins.sass:16]: add EOL

**Review by kipod — COMMENTED** [review]
Looks OK, as for me...
Try to write more comments in your code...
Thanks

File: src/components/layouts/sign_up_sp_layout/SignUpSPLayout.stories.tsx (hunk 1 above)
Comment (varvarashcherbyna7) [reply src/components/layouts/sign_up_sp_layout/SignUpSPLayout.stories.tsx]: thanks i will do that

File: src/pages/index.tsx (hunk 2 above)
Comment (varvarashcherbyna7) [reply src/pages/index.tsx]: this is still a test page

File: src/pages/sign_up/coach_student.tsx (hunk 3 above)
Comment (varvarashcherbyna7) [reply src/pages/sign_up/coach_student.tsx]: ok)

File: src/pages/sign_up/coach_student.tsx (hunk 4 above)
Comment (varvarashcherbyna7) [reply src/pages/sign_up/coach_student.tsx:17]: ok)

**Pull request merged**